app/operations/tests/start_operation.rb

```diff
@@ -0,0 +1,16 @@
+module Tests
+  class StartOperation
+    def initialize(test, current_user)
```

Comment:
For this class it's just a user, not `current_user`. Technically you can instantiate it for any user, so the name should reflect semantics.